server/lorax_server/utils/adapter.py

```diff
@@ -27,8 +28,10 @@ def load_module_map(model_id, adapter_id, adapter_source, weight_names):
     config_path = get_config_path(adapter_id, adapter_source)
     adapter_config = LoraConfig.from_pretrained(config_path)
     if adapter_config.base_model_name_or_path != model_id:
```

Comment:
Would it make sense to download the config of the base model behind adapter and compare with the base model served ? If it's the same model arch just warn and if the arch is different raise an error ?
Could be problematic when someone tries to load bert adapter to llama

Reply:
Good point! Let me add that.

Reply:
Updated.